src/App.js

```diff
@@ -0,0 +1,51 @@
+// import logo from './logo.svg';
+// import './App.css';
+import styled from "styled-components"
+import Header from './components/header/header.jsx';
+import VideoCard from "./components/videocard/videocard.jsx"
+//import "./css/index.css"
+import 'bootstrap/dist/css/bootstrap.min.css';
+import js from "../src/data/yt";
+
+
+const GridWrapper = styled.div`
+display:grid;
+gap:1rem;
+grid-template-columns:repeat(auto-fit, minmax(300px, 1fr))
+`
```

Comment:
There is something weird with the indentation please take a look and if needed configure your editor to fix it automatically.